packages/nx/src/command-line/init/implementation/dot-nx/nxw.ts

```diff
@@ -52,37 +59,70 @@ function ensureDir(p: string) {
   }
 }
 
+function getCurrentInstallation(): PackageJson {
+  try {
+    return require(installationPath);
+  } catch {
+    return {
+      name: 'nx-installation',
+      version: '0.0.0',
+      devDependencies: {},
+    };
+  }
+}
+
+function performInstallation(
+  currentInstallation: PackageJson,
+  nxJson: NxJsonConfiguration
+) {
+  fs.writeFileSync(
+    installationPath,
+    JSON.stringify({
+      ...currentInstallation,
```

Comment:
We shouldn't spread current installation here, the file is meant to be ephemeral / generated by us and not checked in to git so it doesn't make sense to keep changes that are on disk to me. That may make people want to check it in / etc

Reply:
That makes sense 👍🏻